README.md

```diff
@@ -3,31 +3,12 @@
     <img alt="Qiskit" src="https://qiskit.org/images/qiskit-logo.png" width="70" />
   </a>
 </p>
+
 <h1 align="center">
-  Qiskit's website
+  Welcome to <a href="https://qiskit.org">Qiskit.org</a>
```

Comment:
Since we're working with a Markdown file here, I think it might be better to use Markdown links:

```suggestion
  Welcome to [Qiskit.org](https://qiskit.org)
```

Reply:
@eddybrando I tried this, but since this section is centered with `align="center`, we can't use markdown _inside_ of the HTML tags.

This is what happens:
![Screenshot 2023-05-08 at 6 11 45 AM](https://user-images.githubusercontent.com/6276074/236833668-5453fbdc-8cd0-41c6-b094-5ed5d1fa47a7.png)

We don't necessarily have to keep things centered? What do you think?

Reply:
I think it looks nice centered. We could change it, but maybe it makes sense to change it only if we're already making a bigger "design" change on the readme page, which I believe is out of the scope of this PR.